game-headed/build.gradle

```diff
@@ -13,18 +13,18 @@ mainClassName = 'org.triplea.game.client.HeadedGameRunner'
 ext {
     releasesDir = file("$buildDir/releases")
 }
-if (JavaVersion.current() >= JavaVersion.VERSION_11) {
-    apply plugin: 'org.openjfx.javafxplugin'
-    javafx {
-        modules = [ 'javafx.controls', 'javafx.graphics', 'javafx.fxml' ]
-    }
-}
+
+apply plugin: 'org.openjfx.javafxplugin'
```

Comment:
This should no longer be needed as the plugins is already applied above.
Previously this was just so that the plugin only gets applied when using java 11 or greater

Reply:
Makes me wonder why the first line was needed to begin with: https://github.com/triplea-game/triplea/pull/4876/files/375c3fa79687c79628e4e9c5089d3a31c6608764#diff-6d61fe3772b3e0ac9261f3c5ec4c22a0L5

Does not matter, suffice it to say I'm not a fan of gradle build files as they strike me as magic groovy configuration.

Reply:
Well I had to build this weird workaround because starting with Java 11 JavaFX is no longer part of the standard library.
There was a specific reason (I don't recall) why the existing setup didn't work anymore, so I had to somehow make the gradle build work when JavaFX is part of the standard library and when JavaFX has to be treated as independent dependency.
The JavaFX plugin that made all of this work however only worked for Java 11+, thus this ugly workaround.
I'm actually quite relieved to see this weird config removed ^^

Reply:
And the first line is just gradle specification, all plugins must be declared there.
(I re-read your question and realized what you meant)